command/aarelayer/service/aa_config.go

```diff
@@ -0,0 +1,59 @@
+package service
+
+import (
+	"embed"
+	"encoding/json"
+	"strings"
+
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+//go:embed config/*
+var configFile embed.FS
+
+type AAConfig struct {
+	AllowContractCreation bool     `json:"allowContractCreation"`
+	AllowList             []string `json:"allowList"`
+	DenyList              []string `json:"denyList"`
+}
+
+func (c *AAConfig) IsValidAddress(address *types.Address) bool {
+	if address == nil {
+		return c.AllowContractCreation
+	}
+
+	addressStr := strings.TrimPrefix(address.String(), "0x")
+
+	for _, v := range c.DenyList {
+		if addressStr == strings.TrimPrefix(v, "0x") {
+			return false
+		}
+	}
+
+	if len(c.AllowList) == 0 {
+		return true
+	}
+
+	for _, v := range c.AllowList {
+		if addressStr == strings.TrimPrefix(v, "0x") {
+			return true
+		}
+	}
+
+	return false
+}
+
+func GetConfig() (*AAConfig, error) {
```

Comment:
I would prefer to declare this as:
```
func DefaultConfig() *AAConfig {
  return &AAConfig{}
}
```
following the standard in the repo. We can avoid returning an error in the function, the unmarshaling and the embed. I would still keep the embed in a test to check that we can decode a json file (user input) to the config.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1252/commits/55ed79ac314c5508b8b842c188aa5a42f7849f06